package.json

```diff
@@ -13,15 +13,18 @@
   "main": "server/main.js",
   "dependencies": {
     "babel-core": "6.3.17",
+    "bluebird": "^3.1.4",
```

Comment:
I would like to avoid relying on semantic versioning here and fix the dependencies' versions to the revision. Can you take off the `^` from the three new dependencies?